src/Microsoft.AspNetCore.SignalR.Common/Protocol/HandshakeProtocol.cs

```diff
@@ -164,6 +188,7 @@ public static bool TryParseResponseMessage(ref ReadOnlySequence<byte> buffer, ou
             }
         }
 
+
```

Comment:
remove

Reply:
still a blank line here that isn't needed